extensions/blocks/social-previews/modal.js

```diff
@@ -90,7 +90,9 @@ export default withSelect( ( select, props ) => {
 	return {
 		post: getCurrentPost(),
 		title: getEditedPostAttribute( 'title' ),
-		description: getEditedPostAttribute( 'excerpt' ) || getEditedPostAttribute( 'content' ),
+		description:
+			getEditedPostAttribute( 'excerpt' ) ||
+			getEditedPostAttribute( 'content' ).split( '<!--more-->' )[ 0 ],
```

Comment:
Should we support custom "Read More" links as well? Things like this:

`<!--more Click here to find out more!-->`

![image](https://user-images.githubusercontent.com/426388/90627913-f7a57980-e21c-11ea-89c3-2278679b1b62.png)

Reply:
The backend doesn't support it this way but I think we should do that an align them later.